components/infobox/commons/infobox_league.lua

```diff
@@ -0,0 +1,284 @@
+            )
+            :fcell(Cell  :new('Organizer')
+                        :options({})
+                        :content(
+                            unpack(League:_createOrganizers(args))
+                        )
+                        :make()
+            )
+            :cell('Sponsor(s)', args.sponsor)
+            :fcell(Cell  :new('Mode')
+                        :options({})
+                        :content(args.mode)
+                        :variables({key = 'mode', value = args.mode})
+                        :make()
+            )
+            :cell('Server', args.server)
+            :fcell(Cell  :new('Type')
+                        :options({})
+                        :content(args.type)
+                        :categories(
+                            function(_, ...)
+                                local value = select(1, ...)
+                                value = tostring(value):lower()
+                                if value == 'offline' then
+                                    infobox:categories('Offline Tournaments')
+                                elseif value == 'online' then
+                                    infobox:categories('Online Tournaments')
+                                else
+                                    infobox:categories('Unknown Type Tournaments')
+                                end
+                            end
+                        )
+                        :variables({key = 'tournament_type', value = args.type})
+                        :make()
+            )
+            :cell('Location', League:_createLocation({
+                region = args.region,
+                country = args.country,
+                location = args.city or args.location
+            }))
+            :cell('Venue', args.venue)
+            :fcell(self:createPrizepool(args):make())
+            :fcell(Cell :new('Date')
+                        :options({})
+                        :content(args.date)
+                        :variables(
+                            {key = 'date', value = self:_cleanDate(args.date)},
+                            {key = 'tournament_date',
+								value = self:_cleanDate(args.edate) or self:_cleanDate(args.date) or ''}
+                        )
+						:make()
+            )
+            :fcell(Cell :new('Start Date')
+                        :options({})
+                        :content(args.sdate)
+                        :variables(
+                            {key = 'sdate', value = self:_cleanDate(args.sdate)}
+                        )
+						:make()
+            )
+            :fcell(Cell :new('End Date')
+                        :options({})
+                        :content(args.edate)
+                        :variables(
+                            {key = 'edate', value = self:_cleanDate(args.edate)}
+                        )
+						:make()
+            )
+            :fcell(self:createTier(args):make())
+    League:addCustomCells(infobox, args)
```

Comment:
should we maybe move this up above prize pool?
from looking at the main wikis current infobox league templates the custom stuff is added above prize pool

Reply:
You sure?

![image](https://user-images.githubusercontent.com/5138348/129860748-40c0c489-ee73-46de-8739-2e119cbdd695.png)

Reply:
I think it's a good new standard to have custom stuff below the shared properties

Reply:
i liked game version up there better (esp. if we have a highlighted tier (tier 1/2 EPT events)) set), but i can live withit being below the tier
![Screenshot 2021-08-18 13 22 48](https://user-images.githubusercontent.com/75081997/129889953-101ae0c8-2852-4503-9a09-8b39f423cabb.png)

Reply:
Maybe we need to build some kind of sorting function later on. For now, I think placing it above tier is a bit arbitrary, every week will have a different order, perfection is impossible here.